back/src/main/java/ftn/rbs/madagascar_hub/security/jwt/TokenUtils.java

```diff
@@ -0,0 +1,300 @@
+package ftn.rbs.madagascar_hub.security.jwt;
+
+import java.util.Date;
+
+import ftn.rbs.madagascar_hub.models.User;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import jakarta.servlet.http.HttpServletRequest;
+
+// Utility klasa za rad sa JSON Web Tokenima
+@Component
+public class TokenUtils {
+
+	@Value("Certificate_manager")
+	private String APP_NAME;
+
+	@Value("somesecret")
```

Comment:
Move this to environment variables on your computer.

In TokenUtils:
@Value("${token.secret}")

In application.properties
{token.secret=${SECRET}

Reply:
Oh, thanks for the review! I'm going to check it now!